cpmonitor/models.py

```diff
@@ -1,10 +1,71 @@
 from django.db import models
+from django.core.validators import MaxValueValidator, MinValueValidator
 
 # Note PEP-8 naming convetions for class names apply. So use the singular and CamelCase
 
 
+### Lookup-entities (shared among all cities, entered by admins)
+
 class Sector(models.Model):
     name = models.CharField(max_length=100, unique=True)
+    ord = models.IntegerField(unique=True)
```

Comment:
What's with the ord fields?  What are they for?

Reply:
ord = ordinal number to sort sectors.
Usecase:  Display all tasks grouped by sectors in a meaningful order.

Reply:
Hmm your explanation is confusing:  Why? Because I think what you mean is 

Usecase: Display all TaskCategories grouped by sectors in a meaningful oder.

But that would explain the ord field on TaskCategory and not on Sector. 

Maybe a better way to phrase my suspicion is what is the scenario where ordering by name and possibly sector etc is not better than an additional numeric field?

That is shouldn't we order by fields that have meaning?  Instead of some arbitrary number?

Something akin to display all critical tasks grouped by sector (each sector in alphabetic order) and within each sector all tasks sorted by task category (again in alphabetic order) and then by name of task.

Reply:
So do you have an actual use case where that clearly is a lot worse than what you are proposing?  Because certainly people won't know the magic number by heart. So if I'm looking for something in a list ordered by magic number it will much harder than if I'm looking for something by name in a list sorted by name ;-)